.github/workflows/sync-docs-from-cli-repo.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           npm run build
 
       - name: Create a pull request
-        if: ${{ github.event.pull_request.merged || github.event_name == 'workflow_dispatch' }}
```

Comment:
🪓 As another note for reviewers, I believe this was meant've been triggered but on it's own YAML will not evaluate boolean expressions.

GitHub Actions might expect syntax like this instead:

```diff
- if: ${{ github.event.pull_request.merged || github.event_name == 'workflow_dispatch' }}
+ if: ${{ github.event.pull_request.merged == 'true' || github.event_name == 'workflow_dispatch' }}
```

But this is not clear to me.